src/Controllers/MyRadio/login.php

```diff
@@ -16,7 +16,7 @@
  */
 
 if (isset($_SESSION['memberid'])) {
-    URLUtils::redirect(Config::$base_url);
+    URLUtils::redirectWithMessage("/", "", "It seems you were trying to login while still being logged in, if that is not what happened, please contact the Computing Team.");
```

Comment:
It goes to the menu (/myradio or /myury-dev etc), since redirect() is a module/action relative function.